app/routes/report/report.js

```diff
@@ -105,7 +105,7 @@ router.route('/')
   .get(async (req, res) => {
     let {
       query: {
-        paginated, limit, offset, sort, project, states, role, searchText, keyVariant, matchingThreshold, category, approvedTherapy, kbVariant, disease, relevance, context, status, reference, sample, evidenceLevel, matchedCancer, pmidRef, variantType,
+        paginated, limit, offset, sort, project, states, role, searchText, keyVariant, matchingThreshold, kbVariant,
```

Comment:
removing these doesn't have any effect on other views in the front end?

Reply:
These are kb_matched_statements columns and not used in the front end.